cypress/integration/UserCanLogInToApplication.spec.js

```diff
@@ -0,0 +1,23 @@
+describe('User can log in to application', () => {
+  beforeEach(() => {
```

Comment:
Don't think you need beforeEach here since you only have one it block. You can just put it inside of the itblock w/o beforeEach.

Reply:
Check again code is pushed up properly